test/functional/p2p_sendheaders.py

```diff
@@ -255,25 +261,25 @@ def test_null_locators(self, test_node, inv_node):
         tip = self.nodes[0].getblockheader(self.nodes[0].generate(1)[0])
         tip_hash = int(tip["hash"], 16)
 
-        inv_node.check_last_announcement(inv=[tip_hash], headers=[])
-        test_node.check_last_announcement(inv=[tip_hash], headers=[])
+        inv_node.check_last_inv_announcement(inv=[tip_hash])
```

Comment:
Calling the method with a named argument isn't strictly necessary, since the method only takes a single argument now, but it does no harm.